perfmetrics/scripts/fio/fio_metrics.py

```diff
@@ -189,6 +189,23 @@ def _load_file_dict(self, filepath) -> Dict[str, Any]:
       raise NoValuesError(f'JSON file {filepath} returned empty object')
     return fio_out
 
+  def _get_global_ramp_time(self, out_json):
```

Comment:
Don't we need to add unit test for these methods?

Reply:
added, thanks!